### DISCUSSION TITLE
日本自由行攻略(2023Final)

### DISCUSSION THREAD

**Discussion by giscus** [lc4t/lc4t.github.io#17] [Announcements]
# 日本自由行攻略(2023Final)

日本一人自由行攻略，绝对有你不曾关注到的细节

https://blog.sakanano.moe/posts/2024_japan_travel_with_myself/

**Comment by KyleChang-zkh** [comment]
非常详细的攻略，感谢LZ！
另外酒店推荐中的APA HOTEL，是摆放日本否认南京大屠杀和慰安妇存在的右翼书籍，并拒绝撤下的酒店。提醒一下大家。

**Reply by lc4t** [reply]
好，在写的时候不了解这些，大家谨慎选择~

**Comment by dftba-fun** [comment]
感谢分享
